src/components/tag/MeeteamTag.tsx

```diff
@@ -13,8 +17,13 @@ const MeeteamTag = ({ tags }: IMeeteamTag) => {
 	const [tagList, setTagList] = useState<string[]>([]);
 	const [isTouched, setIsTouched] = useState<boolean>(false);
 	const [isDropdownVisible, setIsDropdownVisible] = useState<boolean>(false);
-	const options = ['UI/UX', 'GUI', 'CX', 'BI', 'Figma', 'React', 'Spring', 'Node.js'];
 	const dropdownRef = useRef<HTMLDivElement | null>(null);
+	const keywordTag = useDebounce(tagItem, 500);
+
+	const { data, isLoading } = useQuery({
```

Comment:
```suggestion
	const { data, isSuccess } = useQuery({
```

!isLoading 사용하시려면 오히려 isSuccess를 사용하는게 좋을 것 같습니다!

Reply:
구체적으로 좋은 이유가 있을까요??

Reply:
지금은 없긴한데 추후에 로딩 컴포넌트를 추가하는 거는 어떻게 생각하세요??

Reply:
좋습니다~! 스켈레톤 UI 말씀하시는건가요?? 저는 이거는 구인글 게시판 페이지에는 무조건 있으면 좋겠다고 생각했습니당!!!!!

Reply:
저는 현재 로딩에 필요한 로직이 없길래 isLoading===isSuccess라서 변경하는게 나을거라고 생각했는데, 만약 로딩 컴포넌트가 추가하게 된다면 isLoading(로딩시), isSuccess(성공시) 이렇게 같이 사용하는게 좋을것같습니다!

Reply:
넵넵! 지금은 없는데 제가 추가할 계획을 하고 있어서 일단은 isLoading으로 진행할 것 같습니다!

성공이 아닌 로딩 중인 상태에서 로딩중이라는 사실을 사용자에게 인지시키기 위해서 입니다!!
일단 이 부분은 급하지 않아서 다른 작업 후 적용할 계획입니다!!